src/containers/App.jsx

```diff
@@ -102,12 +101,13 @@ const App = ({
   }, [hasError, isFetching, status])
 
   useEffect(() => {
-    if (isReady) {
-      if (isFlagshipApp() && webviewIntent) {
-        webviewIntent.call('hideSplashScreen')
-      }
-    }
-  }, [webviewIntent, isReady])
+    window.addEventListener(
```

Comment:
Why is this working? Wouldn't it be possible that `load` is called before the first React render and so before this hook happens?